Tests/BgsDataTests.cs

```diff
@@ -20,15 +20,12 @@ public void start()
         }
 
         [TestMethod]
-        [DeploymentItem("bgsFaction.json")]
         public void TestFaction()
         {
             string dateTimeStringFormat = "yyyy'-'MM'-'dd'T'HH':'mm':'ss.fff'Z'";
 
             // Test factions data
-            string jsonString = System.IO.File.ReadAllText("bgsFaction.json");
-            JObject response = JsonConvert.DeserializeObject<JObject>(jsonString);
-
+            JObject response = DeserializeJsonResource<JObject>(Tests.Properties.Resources.bgsFaction);
```

Comment:
This is a bit verbose and a bit of a pain point for writing new tests. 

I'd probably go with `using Tests.Properties.Resources`, then see if we can get the compiler to infer the `<JObject>`.